buddyboss/buddyboss-platform
# Edit Activity Popup scroll improvement

### All Submissions:


Fixes #1466 

### How to test the changes in this Pull Request:

1. Edit any activity with long content in it.
2. Check the scrolling behavior

### Proof Screenshots or Video

https://prnt.sc/ugxzb1

### Other information:
